contentful/export-processor/content-types/recommendation-intro.js

```diff
@@ -8,10 +6,10 @@ export default class RecommendationIntro {
   slug;
 
   constructor({ fields, sys }) {
-    this.content = MapContent(fields.content);
-    this.header = fields.header.fields.text;
-    this.id = sys.id;
-    this.maturity = fields.maturity;
-    this.slug = fields.slug;
+      this.content = fields.content;
+      this.header = fields.header.fields.text;
```

Comment:
```suggestion
      this.header = fields.header?.fields.text;
```

Reply:
Just playing devil's advocate... but if we're making header nullable because somewhere in the content there's a missing header (and granted, it's not a required field on Contentful)... is that what we want? I think this only errored because you're running it on dev and we have bad content in there that we've used for testing (see kg-bad-intro for one for eg). 

This error doesn't happen on prod, which is good because we don't want headerless recommendation intros. So it's a bit of a philosophical question of what are we testing for? This might not be the best way to pick up bad content, or we might want to actually prevent bad content from existing in the first place by requiring header in the content model, but this change seems like pointing down the road of making everything human-generated field nullable so we can get to the tests regardless of how not-prod-worthy our dev content is, but if we do that we need to make sure the rest of the tests can catch these nulls and tell us where the error is ie the content or the service. I'm not confident in what cy.get(<element>).contains(null) is going to do, it might just bork Cypress altogether before it runs anything and we won't get any good info from that error.

Reply:
This is the error we get for an undefined header. We shouldn't encounter it but I'll make a note on the ticket about exception handling to include something better.
![image](https://github.com/user-attachments/assets/6e6173f7-3f61-465e-b678-7fe5ce3d5c08)